functions/__sf_section_exit_code.fish

```diff
@@ -0,0 +1,36 @@
+# TODO: Test me
+# Exit-code
+#
+
+function __sf_section_exit_code -d "Shows the exit code from the previous command."
+    # ------------------------------------------------------------------------------
+    # Configuration
+    # ------------------------------------------------------------------------------
+
+    # --------------------------------------------------------------------------
+    # | SPACEFISH_HOST_SHOW | show hostname on local | show hostname on remote |
+    # |---------------------+------------------------+-------------------------|
+    # | false               | test_me                | test_me                 |
+    # | always              | test_me                | test_me                 |
+    # | true                | test_me                | test_me                 |
+    # | needed              | test_me                | test_me                 |
+    # --------------------------------------------------------------------------
+
+    __sf_util_set_default SPACEFISH_EXIT_CODE_SHOW SPACEFISH_EXIT_CODE_SHOW false
+    __sf_util_set_default SPACEFISH_EXIT_CODE_PREFIX SPACEFISH_EXIT_CODE_PREFIX ""
+    __sf_util_set_default SPACEFISH_EXIT_CODE_SUFFIX SPACEFISH_EXIT_CODE_SUFFIX " "
+    __sf_util_set_default SPACEFISH_EXIT_CODE_SYMBOL SPACEFISH_EXIT_CODE_SYMBOL "✘"
+    __sf_util_set_default SPACEFISH_EXIT_CODE_COLOR SPACEFISH_EXIT_CODE_COLOR "red"
+
+    # ------------------------------------------------------------------------------
+    # Section
+    # ------------------------------------------------------------------------------
+
+    [ $SPACEFISH_EXIT_CODE_SHOW = false; or $status -eq 0 ]; and return # I hope this works. Todo: Please test this funky line. Fish uses $status instead of $RETVAL.
```

Comment:
I would suggest using [`$sf_exit_code`](https://github.com/matchai/spacefish/blob/master/fish_prompt.fish#L3) (which was used [in char](https://github.com/matchai/spacefish/blob/master/functions/__sf_section_char.fish#L24))
`$status` will have the status code of the last command, which in thais case would be the last `__sf_util_set_default`. 😉